packages/transaction-controller/src/types.ts

```diff
@@ -17,6 +17,7 @@ import type { Hex } from '@metamask/utils';
  * @property transaction - Underlying Transaction object
  * @property transactionHash - Hash of a successful transaction
  * @property blockNumber - Number of the block where the transaction has been included
+ * @property baseFeePerGas - The estimated block baseFeePerGas that will be burned. Introduced in EIP 1559. Value in hex wei
```

Comment:
It's hard to add detail to these going forward without `.` so I'm wondering if we should start using `.` going forward and add to the other properties for consistency?

Reply:
I'm always unsure, but now that you raised it I vote for going forward using `.`.